skymp5-server/cpp/lib_espm/espm.h

```diff
@@ -779,6 +781,232 @@ class GMST : public RecordHeader
   Data GetData(CompressedFieldsCache& compressedFieldsCache) const noexcept;
 };
 static_assert(sizeof(GMST) == sizeof(RecordHeader));
+
+struct Effects
+{
+public:
+  Effects(){};
+  Effects(const RecordHeader* parent);
+  const espm::RecordHeader* parent = nullptr;
+
+  struct Effect
+  {
+    uint32_t effectId = 0; // EffectID MGEF
```

Comment:
```suggestion
    uint32_t effectId = 0; // Corresponding MGEF record id
```

I think that would be a cleaner comment. Maybe form id. Maybe also renaming it to `effectFormId`. Or whatever